KnapsackUtilsCython.pyx

```diff
@@ -0,0 +1,216 @@
+import logbook
+import math
+import time
+import typing
+import cython
```

Comment:
cython should prob be in requirements.txt as well, don't think its there yet right? Unless it has a special install?

Reply:
added